src/components/form-create-account/form-create-account.js

```diff
@@ -0,0 +1,268 @@
+      <div className="columns">
+        <div className="column is-half">
+          <TextInput
+            label={"Adresa de email"}
+            inputProps={{
+              name: "emailAddress",
+              id: "emailAddress",
+              onChange: input_OnChange,
+              type: "email",
+              required: "true"
+            }}
+          />
+        </div>
+      </div>
+
+      <PasswordWithConfirm onChange={password_OnChange} />
+
+      <div className="columns">
+        <div className="column is-2">
+          <Select
+            label={"Varsta"}
+            selectProps={{
+              name: "ageRange",
+              id: "ageRange",
+              onChange: input_OnChange
+            }}
+            options={ageOptions}
+          />
+        </div>
+        <div className="column is-2">
+          <Select
+            label={"Gen"}
+            selectProps={{
+              name: "gender",
+              id: "gender",
+              onChange: input_OnChange
+            }}
+            options={genderOptions}
+          />
+        </div>
+      </div>
+      <div className="columns">
+        <div className="column">
+          <Select
+            label="Ai condiții de sănătate preexistente?"
+            description="Spune-ne dacă suferi de anumite boli cronice, diabet, hipertensiune etc. "
+            selectProps={{
+              name: "existingHealthConditions",
+              id: "existingHealthConditions",
+              onChange: input_OnChange
+            }}
+            options={illnesesOptions}
+          />
+        </div>
+      </div>
+      <div className="columns">
+        <div className="column">
+          <RadioList
+            label="În ultima perioada ai fost in:"
+            onChange={optiuniUltimaPerioada_OnChange}
+            options={lastPeriodInformationOptions}
+          />
+        </div>
+      </div>
+      <div className="columns">
+        <div className="column is-half">
+          <Select
+            label="Alegeți țara în care vă aflați"
+            selectProps={{
+              name: "country",
+              id: "country",
+              onChange: country_OnChange
+            }}
+            options={countriesList}
+          />
+        </div>
+        <div className="column is-half">
+          <Select
+            label="Alegeți orașul în care vă aflați"
+            selectProps={{
+              name: "city",
+              id: "city",
+              onChange: input_OnChange
+            }}
+            options={cities}
+          />
+        </div>
+      </div>
+      <div className="columns">
+        <div className="column">
+          <label className="checkbox">
+            <input
+              type="checkbox"
+              required
+              name="acceptConditions"
+              id="acceptConditions"
+              onChange={input_OnChange}
+            />
+            &nbsp;&nbsp;&nbsp; Prin această bifă îți exprimi acordul ca datele
```

Comment:
As sugera sa fie intr-o constanta treaba asta, care sa fie setata ca defaultProp astfel incat sa fie as customizable as we can make it

Reply:
Agreed - will move it there